internal/pkg/pkg.go

```diff
@@ -167,7 +168,10 @@ func New(fs filesys.FileSystem, path string) (*Pkg, error) {
 // A nil value represents an implicit package.
 func (p *Pkg) Kptfile() (*kptfilev1.KptFile, error) {
 	if p.kptfile == nil {
-		kf, err := ReadKptfile(types.DiskPath(p.UniquePath.String()))
+		kf, err := ReadKptfile(paths.FileSystemPath{
```

Comment:
Any reason for introducing this wrapper over earlier API `ReadKptfile(fsys, path)` ?